modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -175,29 +312,29 @@ For example:
 `$ curl -H "Authorization: Bearer [YOUR_ACCESS_TOKEN]" https://anypoint.mulesoft.com/accounts/api/me`
 
 [[sharing-resources]]
-== Sharing Organization Resources
+== Share Business Group Resources
 
-All API versions and CloudHub environments that you create in an organization are accessible only to users within the organization.
+All API versions and CloudHub environments that you create in a business group are accessible only to users within the business group.
 
-If you want to share resources with a user, you have to invite the user to join your organization, and the user must accept the invitation and join the organization you sent the invitation from. See xref:users.adoc#inviting-users[Inviting Users] for more information.
+If you want to share resources with a user, you have to invite the user to join your business group, and the user must accept the invitation and join the business group you sent the invitation from. See xref:users.adoc#inviting-users[Inviting Users] for more information.
 
 [NOTE]
 --
-Because the organization name (often the company name) is not necessarily unique, it is not sufficient for the invited user to join the organization and use the company name associated with the root organization. The domain name you set in the organization information is what distinguishes your organization from other organizations.
+Because the business group name (often the company name) is not necessarily unique, it is not sufficient for the invited user to join the business group and use the company name associated with the root business group. The organization domain name you set in the business group information is what distinguishes your business group from other business groups.
```

Comment:
```suggestion
Because the business group name (often the company name) isn't necessarily unique, it isn't sufficient for the invited user to join the business group and use the company name associated with the root business group. The organization domain name you set in the business group information is what distinguishes your business group from other business groups.
```